projects/canopy/src/styles/grid.scss

```diff
@@ -49,6 +49,53 @@ $columns: 12;
   max-width: 100%;
 }
 
+@mixin lg-grid-alignment($breakpoint) {
+  // Alignment and distribution
+
+  .start-#{$breakpoint} {
```

Comment:
Looks good but could you add the `lg-` prefix to the classes please?

Reply:
Done